### PR TITLE
zero: Merge NFC Hal From A8 2016 Oreo (A810SKSU2CRG3)

### PR DESCRIPTION
**Pull request by enesuzun2002**

### CODE REVIEW

File: zero-common/zero-common-vendor.mk [hunk 1]
```diff
@@ -377,14 +375,20 @@ PRODUCT_COPY_FILES += \
...
     vendor/samsung/zero-common/proprietary/vendor/lib/libLifevibes_lvvetx.so:system/vendor/lib/libLifevibes_lvvetx.so \
     vendor/samsung/zero-common/proprietary/vendor/lib/libOpenCL.so:system/vendor/lib/libOpenCL.so \
     vendor/samsung/zero-common/proprietary/vendor/lib/libOpenCv.so:system/vendor/lib/libOpenCv.so \
+    vendor/samsung/zero-common/proprietary/vendor/lib/vendor.samsung.hardware.nfc@1.0.so:system/vendor/lib/vendor.samsung.hardware.nfc@1.0.so \
     vendor/samsung/zero-common/proprietary/vendor/lib/mediadrm/libdrmclearkeyplugin.so:system/vendor/lib/mediadrm/libdrmclearkeyplugin.so \
     vendor/samsung/zero-common/proprietary/vendor/lib/mediadrm/libwvdrmengine.so:system/vendor/lib/mediadrm/libwvdrmengine.so \
     vendor/samsung/zero-common/proprietary/vendor/lib64/egl/libGLES_mali.so:system/vendor/lib64/egl/libGLES_mali.so \
     vendor/samsung/zero-common/proprietary/vendor/lib64/hw/vulkan.exynos5.so:system/vendor/lib64/hw/vulkan.exynos5.so \
+    vendor/samsung/zero-common/proprietary/vendor/lib64/hw/nfc_nci.default.so:system/vendor/lib64/hw/nfc_nci.default.so \
```
Comment (npjohnson) [inline zero-common/zero-common-vendor.mk:389]: This isn't script generated, which we require. If it was, the alphabetization would've happened.

**Review by npjohnson — CHANGES_REQUESTED** [review]
Does the current maintainer (CC: @Usaamah99) agree with this change?

File: zero-common/zero-common-vendor.mk (hunk 1 above)
Comment (enesuzun2002) [reply zero-common/zero-common-vendor.mk:389]: yes it is not, i have added them myself to list. Also yes @Usaamah99 know about this

**Comment by enesuzun2002** [comment]
i will create new one right now this one has some issues

**Comment by enesuzun2002** [comment]
it is fine and can be merged

**Comment by npjohnson** [comment]
@enesuzun2002 @Usaamah99 said this is NOT to be merged right now. When he approves it, we can merge.

**Comment by npjohnson** [comment]
Also, the comments aren't fixed. This wasn't script gen'd.

**Pull request closed without merging**